frontend/src/lib/lemon-ui/Tooltip/Tooltip.tsx

```diff
@@ -136,6 +156,13 @@ export function Tooltip({
                             style={{ ...transitionStyles }}
                         >
                             {typeof title === 'function' ? title() : title}
+                            {docLink && (
+                                <p className={`mb-0 ${title ? 'mt-1' : ''}`}>
+                                    <Link to={docLink} target="_blank" className="text-xs">
+                                        Read the docs
+                                    </Link>
+                                </p>
+                            )}
```

Comment:
If `title` was provided, we should probably just render as the small `docs` icon to the right rather than rendering **under** it + "Read the docs" label.

Reply:
I like that it's more explicit for now